content/association/_index.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "Association"
++++
+
+> Note:
+>
+> This page is a draft and is subject to change. The association is not yet registered
+> and we are still considering options on how and where it should be. To help us
+> develop these statutes or the association you can join us in the
+> [Biweekly Nordic RSE meeting](/communities/join/#bi-weekly-meetings).
+> 
```

Comment:
```suggestion
> 
> Our suggestion at the moment is to register the association in Finland, since we know the relevant laws.
> This is not necessarily an informed decision and we would welcome input about the possibility of registering
> in a different Nordic country.
>
> It should be possible for members, board members and the chair to not be citizens of the registration country.
> Other than that, the registration process should be as straightforward as possible.
>
```

Reply:
@rantahar Shouldn't we commit your proposed change and merge this PR?